### PR TITLE
Add request body support to h2load

### PR DESCRIPTION
**Pull request by JamesNK**
Fixes https://github.com/aspnet/Benchmarks/issues/825

### CODE REVIEW

**Comment by JamesNK** [comment]
Note: completely untested!

Will add some tests that use this to the gRPC repo.

**Comment by sebastienros** [comment]
Note you can use @responsefile if the body is big enough, so you can externalize it from the command line

**Comment by JamesNK** [comment]
Part of https://github.com/grpc/grpc-dotnet/issues/6

File: src/BenchmarksClient/Workers/H2LoadWorker.cs [hunk 1]
```diff
@@ -51,14 +58,26 @@ private void InitializeJob()
         public async Task StartJobAsync(ClientJob job)
         {
             _job = job;
+
+            if (job.ClientProperties.TryGetValue("RequestBody", out var requestBodyText))
```
Comment (sebastienros) [inline src/BenchmarksClient/Workers/H2LoadWorker.cs]: I would just put this block in `StartProcess`

You get bonus points if you handle it on wrk, wrk2 and bombardier ;)

**Pull request merged**